hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/RpcServer.java

```diff
@@ -313,6 +315,10 @@ public void onConfigurationChange(Configuration newConf) {
     if (scheduler instanceof ConfigurationObserver) {
       ((ConfigurationObserver) scheduler).onConfigurationChange(newConf);
     }
+    synchronized (authManager) {
```

Comment:
nit: Add a quick log that we are re-setting the authManager state, probably helps with debugging races later if any..